### PR TITLE
Add Blend and CCS docs.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
## Cute animal pic
_Because everyone likes pictures of animals._

gerbil

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

- https://trello.com/c/PSE6zvXO
- https://trello.com/c/pScapQ9d

## Description
_The commit messages say what you did; this should explain why and how._

- Add docs for Blend
- Add docs for CCS
- Other missing updates from OddSite

## Impacted areas of website
_Provide URLs and/or a description of where to look._

- /open-source/
- /blend/
- /blend/docs/
- /cascading-colors/
- /cascading-colors/docs/

---

## Reviewer tasks:

- [ ] Pulled branch and ran code locally (or viewed on preview deployment, if applicable)
- [ ] Reproduced the issue to review in the browser
- [ ] Reviewed code
- [ ] Updated Trello card (if applicable)

### CODE REVIEW

File: content/open-source/blend.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  - miriam
+  - oddbird
+index: Blend
+links:
+  docs: /blend/docs/
+  source: https://github.com/oddbird/blend
+badges:
+  - name: npm
+    src: https://badge.fury.io/js/%40oddbird%2Fblend.svg
+    url: https://www.npmjs.com/package/@oddbird/blend
```
Comment (jgerigmeyer) [inline content/open-source/blend.md:18]: I note that this isn't actually being used/displayed, but 🤷

**Comment by jgerigmeyer** [comment]
[Add Blend page/docs to OddLeventy](https://trello.com/c/PSE6zvXO/68-add-blend-page-docs-to-oddleventy)

**Comment by jgerigmeyer** [comment]
[Add cascading-color docs to site](https://trello.com/c/pScapQ9d/72-add-cascading-color-docs-to-site)

File: content/open-source/cascading-colors.njk [hunk 2]
```diff
@@ -13,8 +13,12 @@ oss: owner
 date: 2019-09-07
 index: Cascading Colors
 links:
-  docs: https://cascading-colors.netlify.com/
-  source: https://github.com/mirisuzanne/cascading-color-system
+  docs: /cascading-colors/docs/
+  source: https://github.com/oddbird/cascading-color-system
```
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/open-source/cascading-colors.njk:17]: @jgerigmeyer Yeah, that's probably a good idea. Related: should we host the demos on oddbird.net, or use the netlify url for that?

[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply content/open-source/cascading-colors.njk:17]: I'm fine either way -- I thought your argument was that the demo should stay with the tool itself? In which case linking to netlify.

[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/open-source/cascading-colors.njk:17]: I want demo code in the repo for functional testing… but that wouldn't have to be public. The blend demo was never really meant for public display, and the cascading colors demo doesn't provide much new over our oddleventy usage + docs. Maybe we don't need them anywhere?

[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply content/open-source/cascading-colors.njk:17]: Sure, I'm fine with that -- in which case I think this PR is good to go?

**Comment by jgerigmeyer** [comment]
@mirisuzanne I assume you're either addressing or making a new card for https://github.com/oddbird/oddleventy/pull/54#pullrequestreview-435264790?

File: content/open-source/cascading-colors.njk (hunk 2 above)
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/open-source/cascading-colors.njk:17]: I'm still doing final cleanup and review, mostly addressing the badges comment above.

File: content/open-source/blend.md (hunk 1 above)
Comment (mirisuzanne) [reply content/open-source/blend.md:18]: @jgerigmeyer it is now, if you want to review that

File: content/_includes/page/intro.njk [hunk 3]
```diff
@@ -1,7 +1,24 @@
+{%- import 'embed.macros.njk' as embed -%}
```
Comment (jgerigmeyer) [inline content/_includes/page/intro.njk:1]: Why the `{%-` instead of `{%`?

Comment (mirisuzanne) [reply content/_includes/page/intro.njk:1]: I just copy/pasted in this case, didn't think about it.

**Pull request merged**